website/templates/website/documentation_page.html

```diff
@@ -1,6 +1,15 @@
 {% extends "website/base_template.html" %}
 {% load customfilters %}
 
+{% block extra_css %}
+<style type="text/css">
+	.section {
+    padding-top: 50px;
```

Comment:
And, when the text linked to the permalink renders on more than one line, it still has a weird behavior. For instance, see http://dipy.herokuapp.com/documentation/0.12.0.dev/examples_built/contextual_enhancement/#example-contextual-enhancement